src/inc/gcinfoencoder.h

```diff
@@ -110,8 +110,12 @@ struct GcInfoSize
     size_t NumRanges;
     size_t NumRegs;
     size_t NumStack;
+    size_t NumUntracked;
```

Comment:
Can you please make sure that these fields are handled correctly in  `GcInfoSize::operator+=` and `GcInfoSize::Log`